src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -149,14 +147,14 @@ public TrapElvSubsystem() {
     baseMotorOffset1 = 0.0;
     baseMotorOffset2 = 0.0;
 
-    baseMotor1 = new CANSparkMaxSim(TrapElvConstants.BASE_MOTOR1_ID, MotorType.kBrushless);
+    baseMotor1 = new CANSparkMax(TrapElvConstants.BASE_MOTOR1_ID, MotorType.kBrushless);
     baseMotor1.restoreFactoryDefaults();
     baseMotor1.getPIDController().setP(TrapElvConstants.BASE_PID[0]);
     baseMotor1.getPIDController().setI(TrapElvConstants.BASE_PID[1]);
     baseMotor1.getPIDController().setD(TrapElvConstants.BASE_PID[2]);
     baseMotor1.getPIDController().setIZone(TrapElvConstants.BASE_PID[3]);
     baseMotor1.getPIDController().setFF(TrapElvConstants.BASE_PID[4]);
-    TrapElvTab.add("Base Elv PID", baseMotor1.getPIDController());
+    // TrapElvTab.add("Base Elv PID", baseMotor1.getPIDController());
```

Comment:
I suspect this is what was causing the issue. 
 
I didn't make initSendable() well to take into account that getP/I/D() would be making CAN transactions every cycle.

https://github.com/frc6377/crescendo_2024/blob/main/src%2Fmain%2Fjava%2Fcom%2Frevrobotics%2FCANSparkMaxSim.java#L56-L79

Reply:
Want to test it this week?  and make a decision of next steps from that?